From 95927956e8473d4953a9b5b1df36f60ebf1730d9 Mon Sep 17 00:00:00 2001
From: Hongxu Jia <hongxu.jia@windriver.com>
Date: Wed, 18 Oct 2017 10:14:17 +0800
Subject: [PATCH] do not limit the format of key file

Upstream-Status: Inappropriate [wrlinux specific]

Signed-off-by: Hongxu Jia <hongxu.jia@windriver.com>
---
 lib/rpmts.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lib/rpmts.c b/lib/rpmts.c
index 021c406..c5548ce 100644
--- a/lib/rpmts.c
+++ b/lib/rpmts.c
@@ -278,7 +278,7 @@ static int loadKeyringFromFiles(rpmts ts)
 {
     ARGV_t files = NULL;
     /* XXX TODO: deal with chroot path issues */
-    char *pkpath = rpmGetPath(ts->rootDir, "%{_keyringpath}/*.key", NULL);
+    char *pkpath = rpmGetPath(ts->rootDir, "%{_keyringpath}/*", NULL);
     int nkeys = 0;
 
     rpmlog(RPMLOG_DEBUG, "loading keyring from pubkeys in %s\n", pkpath);
-- 
2.34.1

